src/measure_theory/function/lp_space.lean

```diff
@@ -2906,6 +2916,13 @@ rfl
   (to_Lp p μ 𝕜 f : α →ₘ[μ] E) = f.to_ae_eq_fun μ :=
 rfl
 
+lemma to_Lp_inj {f g : C(α, E)} [μ.is_open_pos_measure] [normed_field 𝕜] [normed_space 𝕜 E]
+  (h : to_Lp p μ 𝕜 f = to_Lp p μ 𝕜 g) : f = g :=
+(linear_isometry_bounded_of_compact α E 𝕜).injective (bounded_continuous_function.to_Lp_inj _ h)
```

Comment:
The naming guide reccommends that `_inj` be the name of the `iff` lemma, and to skip the bare implication altogether.